tests/integration/conftest.py

```diff
@@ -177,3 +178,33 @@ def localstack_runtime():
     localstack_started.wait()
     yield
     return
+
+
+@pytest.fixture
+def create_rest_apigw(apigateway_client):
+    rest_api_ids = []
+
+    def _create_apigateway_function(*args, **kwargs):
+        api_id, name, root_id = create_rest_api(apigateway_client, **kwargs)
+        rest_api_ids.append(api_id)
+        return api_id, name, root_id
+
+    yield _create_apigateway_function
+
+    for rest_api_id in rest_api_ids:
+        delete_rest_api(apigateway_client, restApiId=rest_api_id)
+
+
+@pytest.fixture
+def import_apigateway_function(apigateway_client):
```

Comment:
nit: I guess the same could apply to shortening the name of this fixture.. 😅 (don't bother for this PR, though - could be refactored in a future iteration)

Reply:
umpf 😅 yep! made the quick fix